.idea/dictionaries/ovezovs.xml

```diff
@@ -0,0 +1,9 @@
+<component name="ProjectDictionaryState">
+  <dictionary name="ovezovs">
```

Comment:
was the name "ovezovs" used for testing purposes? I think we should another name for it, more descriptive of the dictionary's functionality.

Reply:
I renamed the file to be more descriptive. Thanks!